examples/next/faustwp-getting-started/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "faustwp dev",
     "build": "faustwp build",
+    "generate": "faustwp generate",
```

Comment:
Regarding the human readable naming, I wonder if the CLI command should be just `faust` as in `faust dev`, `faust build`, etc.